components/input-validation-mgt/org.wso2.carbon.identity.input.validation.mgt/src/main/java/org/wso2/carbon/identity/input/validation/mgt/listener/InputValidationListener.java

```diff
@@ -110,62 +118,75 @@ public boolean doPreUpdateCredentialWithID(String userID, Object newCredential,
         if (!isEnable()) {
             return true;
         }
-        return validate(PASSWORD, newCredential.toString(), userStoreManager);
+        return validate(Collections.singletonMap(PASSWORD, newCredential.toString()), userStoreManager);
 
     }
 
     /**
      * Method to validate the values.
      *
-     * @param field             Field to be validated.
-     * @param value             Value to be validated.
+     * @param fieldValueMap     Map of fields and values that need to be validated.
```

Comment:
Add a sample value and meaningful map name

Reply:
Addressed